app/app.js

```diff
@@ -10,8 +10,8 @@ const bodyParser = require('body-parser');
 const dateFilter = require('nunjucks-date-filter');
 
 // Local dependencies
-const config = require('./app/config');
-const locals = require('./app/locals');
+const config = require('./config');
```

Comment:
are you just moving these to be consistent with the other apps?

Reply:
yes, but also the app and server.js were being included in the package even though I didn't whitelist them. But moving them in the app folder doesn't include these files as part of the package

Reply:
sweet